### PR TITLE
Update workflows from template repository

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ <skjnldsv@protonmail.com>

### CODE REVIEW

File: vendor/autoload.php [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 require_once __DIR__ . '/composer/autoload_real.php';
 
-return ComposerAutoloaderInitf0bdd5b9b70328ede264941d59e950bb::getLoader();
+return ComposerAutoloaderInit38d0ef20c0615b2089b379ca9b55f35e::getLoader();
```
Comment (nickvergessen) [inline vendor/autoload.php:7]: Seems you commited the dev dependency?

File: vendor/geoip2/geoip2/.github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+version: 2
```
Comment (nickvergessen) [inline vendor/geoip2/geoip2/.github/dependabot.yml:1]: brings in a lot of dev files which are not meant to ship.
So either the exclude list on packaging needs heavy additions, or we revert all changes on vendor/ and install the dev dependency only on the cs-checker. (I was actually too lazy for that and that is why this app did not use psalm so far

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: vendor/geoip2/geoip2/.github/dependabot.yml (hunk 2 above)
Comment (skjnldsv) [reply vendor/geoip2/geoip2/.github/dependabot.yml:1]: Ahaha, I fell into the trap
I thought that was gonna be an easy task
let me remove php-cs for now

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**